File: 백준/Bronze/25304. 영수증/영수증.cc
#include <iostream>
using namespace std;

int main() {

	int total_cost;
	int kind;
	int price;
	int n;
	int sum = 0;

	cin >> total_cost;
	cin >> kind;

	for (int i = 0; i < kind; i++)
	{
		cin >> price;
		cin >> n;
		sum += (price * n);
	}

	if (total_cost == sum)
	{
		cout << "Yes" << endl;
	}
	else {
		cout << "No" << endl;
	}


	return 0;
}

File: 백준/Bronze/25304. 영수증/영수증.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int totalPrice = sc.nextInt();
        int kindOf = sc.nextInt();
        int sum = 0;

        for (int i = 0; i < kindOf; i++) {
            int price = sc.nextInt();
            int n = sc.nextInt();

            sum += price * n;
        }
        if (sum == totalPrice) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}